controllers/profiles/reconciler_dev.go

```diff
@@ -324,11 +363,114 @@ func mountDevConfigMapsMutateVisitor(flowDefCM, propsCM *v1.ConfigMap) mutateVis
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = make([]v1.VolumeMount, 0)
 			deployment.Spec.Template.Spec.Containers[0].VolumeMounts = volumeMounts
 
+			for keycm, cm := range externalResourceConfigMaps {
+				if !isContained(cm.Name, deployment.Spec.Template.Spec.Volumes) {
+					deployment.Spec.Template.Spec.Volumes = append(deployment.Spec.Template.Spec.Volumes,
+						v1.Volume{
+							Name: cm.Name,
+							VolumeSource: v1.VolumeSource{
+								ConfigMap: &v1.ConfigMapVolumeSource{
+									LocalObjectReference: v1.LocalObjectReference{Name: cm.Name},
+								},
+							},
+						})
+				}
+				if !isContainedVM(getMountPath(keycm), deployment.Spec.Template.Spec.Containers[0].VolumeMounts) {
+					deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts,
+						v1.VolumeMount{
+							Name:      cm.Name,
+							ReadOnly:  true,
+							MountPath: getMountPath(keycm),
+						})
+				}
+			}
 			return nil
 		}
 	}
 }
 
+func getMountPath(name string) string {
+	var mountPath = quarkusDevConfigMountPath
+	if strings.HasSuffix(name, extResCamelSuffix) {
+		return getConfigMountPath(CAMEL)
+	}
+	if strings.HasSuffix(name, extResGenericSuffix) {
+		return getConfigMountPath(GENERIC)
+	}
+	if strings.HasSuffix(name, extResAsyncAPISuffix) {
+		return getConfigMountPath(ASYNCAPI)
+	}
+	if strings.HasSuffix(name, extResOpenAPISuffix) {
+		return getConfigMountPath(OPENAPI)
+	}
+	return mountPath
+}
+
+func loadExternalResourcesConfigMaps(support *stateSupport, workflow *operatorapi.KogitoServerlessWorkflow) map[string]v1.ConfigMap {
+	externalConfigMaps := make(map[string]v1.ConfigMap, 0)
+	for k, val := range workflow.Labels {
+		if strings.HasPrefix(k, extResPrefix) {
+			resource, found := cutPrefix(k, "sw.kogito.kie.org/")
```

Comment:
So you should not need this if you get rid of that const.

Reply:
I cut the prefix only on the label like sw.kogito.kie.org/resource-*